### PR TITLE
Add: stardust-xr terra package guide

### PR DESCRIPTION
**Pull request by Owen-sz**
Waiting for more concrete post-install directions.

### CODE REVIEW

File: src/content/docs/en/anywhere/xr.mdx [hunk 1]
```diff
@@ -10,4 +10,28 @@ import Alert from "../../../../components/Docs/Alert.astro";
...
 </Alert>
 
-Would use [StardustXR](https://stardustxr.org/). Our packaging effort can be found [here.](https://github.com/Ultramarine-Linux/stardustxr)
+We will use [Stardust XR](https://stardustxr.org/). 
+
+All core Stardust XR components are packaged in Terra:
+
+- stardust-xr-armillary
+- stardust-xr-atmosphere
+- stardust-xr-black-hole
```
Comment (nothingneko) [inline src/content/docs/en/anywhere/xr.mdx:19]: do these in `code` or use links to repology/pkgs.org/terra hub in future

File: src/content/docs/en/anywhere/xr.mdx [hunk 2]
```diff
@@ -10,4 +10,28 @@ import Alert from "../../../../components/Docs/Alert.astro";
...
+- [stardust-xr-atmosphere](https://repology.org/project/stardust-xr-atmosphere/versions)
+- [stardust-xr-black-hole](https://repology.org/project/stardust-xr-black-hole/versions)
+- [stardust-xr-comet](https://repology.org/project/stardust-xr-comet/versions)
+- [stardust-xr-flatland](https://repology.org/project/stardust-xr-flatland/versions)
+- [stardust-xr-gravity](https://repology.org/project/stardust-xr-gravity/versions)
+- [stardust-xr-magnetar](https://repology.org/project/stardust-xr-magnetar/versions)
+- [stardust-xr-non-spatial-input](https://repology.org/project/stardust-xr-non-spatial-input/versions)
+- [stardust-xr-protostar](https://repology.org/project/stardust-xr-protostar/versions)
+- [stardust-xr-server](https://repology.org/project/stardust-xr-server/versions)
+- [stardust-xr-telescope](https://repology.org/project/stardust-xr-telescope/versions)
```
Comment (madonuko) [inline src/content/docs/en/anywhere/xr.mdx:27]: pkgs.org comes with installation instructions too, maybe use that instead?

File: src/content/docs/en/anywhere/xr.mdx [hunk 3]
```diff
@@ -10,4 +10,28 @@ import Alert from "../../../../components/Docs/Alert.astro";
...
+
+To install all of these packages:
+
+```
+dnf group install stardust-xr
```
Comment (madonuko) [inline src/content/docs/en/anywhere/xr.mdx:32]:
```suggestion
sudo dnf group install stardust-xr
```

File: src/content/docs/en/anywhere/xr.mdx [hunk 4]
```diff
@@ -10,4 +10,28 @@ import Alert from "../../../../components/Docs/Alert.astro";
   page is subject to change.
 </Alert>
 
-Would use [StardustXR](https://stardustxr.org/). Our packaging effort can be found [here.](https://github.com/Ultramarine-Linux/stardustxr)
+We will use [Stardust XR](https://stardustxr.org/). 
+
+All core Stardust XR components are packaged in Terra:
```
Comment (madonuko) [inline src/content/docs/en/anywhere/xr.mdx:15]: What is Terra?

File: src/content/docs/en/anywhere/xr.mdx [hunk 5]
```diff
@@ -10,4 +10,28 @@ import Alert from "../../../../components/Docs/Alert.astro";
   page is subject to change.
 </Alert>
 
-Would use [StardustXR](https://stardustxr.org/). Our packaging effort can be found [here.](https://github.com/Ultramarine-Linux/stardustxr)
+We will use [Stardust XR](https://stardustxr.org/).
```
Comment (madonuko) [inline src/content/docs/en/anywhere/xr.mdx:13]:
```suggestion
We will provide full support on [Stardust XR](https://stardustxr.org/). 
```

**Review by madonuko — APPROVED** [review]

File: src/content/docs/en/anywhere/xr.mdx [hunk 6]
```diff
@@ -10,4 +10,28 @@ import Alert from "../../../../components/Docs/Alert.astro";
   page is subject to change.
 </Alert>
 
-Would use [StardustXR](https://stardustxr.org/). Our packaging effort can be found [here.](https://github.com/Ultramarine-Linux/stardustxr)
+We will provide full support on [Stardust XR](https://stardustxr.org/).
```
Comment (nothingneko) [inline src/content/docs/en/anywhere/xr.mdx:13]: "for" rather than "on"

remove full

**Review by nothingneko — APPROVED** [review]

**Review by nothingneko — APPROVED** [review]

**Pull request merged**